### DISCUSSION TITLE
Warped Packs and damage

### DISCUSSION THREAD

**Discussion by Semboflorin** [TGWeaver/CDDA-Sky-Islands#20] [General]
The warped packs are great and I understand why you built them the way you did.  So as not to be exploited.  However, there is one flaw I think in the design.  The fact that they fit in their own "aura" location is awesome but it appears that this location is the first line of defense for your character even when none of the packs offer any sort of defense.  So what happens is that when I get attacked the first thing to take damage is the warped pack.  I notice that these packs are VERY fragile.  Since they cannot be repaired and cannot be stowed the only option is to take the time to drop them on the ground before getting into a fight.  If I get hit before I complete this or something is shooting me it generally gets trashed very fast.  I've now lost two harnesses because of this.  Once when I entered a basement and was swarmed and another time I had it shot off me by a roadblock turret (it was already damaged).  I realize that I can just make more but this mechanic makes these things MUCH less useful or desirable.  I generally only bring one when I cannot otherwise carry something normally.  Which basically means the warped animal carrier is all I will now make.

I would suggest that these things either not be the first line of defense, be much more durable, or at the very least be repairable.

**Comment by TGWeaver** [comment]
Huh. In all my testing, and my play since, none of the attacks I've received have ever damaged the warped bags. I guess it just didn't occur to me.
I've uploaded a patch that should fix packs taking hits during combat, but I'm not yet sure how to make them completely damage immune. Should work for now but a future patch may improve this further.

**Comment by TGWeaver** [comment]
This should now be fixed. Let me know if there are any further problems about it.

**Reply by Semboflorin** [reply]
Thank you so much.  So far, while standing in a few different groups of zeds the pack is unharmed.

**Reply by Onkl** [reply]
I started playing after you applied this fix and never had a problem with the warped packs. They work as described.
